assets/js/google-tag.js

```diff
@@ -0,0 +1,15 @@
+---
+layout: blank
+---
+
+window.dataLayer = window.dataLayer || [];
+function gtag(){window.dataLayer.push(arguments);}
+gtag('js', new Date());
+
+// Set Isomer's ga tag
+gtag('config', 'G-3RT85MXN6L');
+
+// Set site specific ga tag
+if ('{{site.google_analytics_ga4}}' && '{{site.google_analytics_ga4}}' != "") {
+  gtag('config', '{{site.google_analytics_ga4}}');
+}
```

Comment:
what happens on sites w/ UA + GA4? they can track both properly right?

Reply:
separately, how will agencies set this `google_analytics_ga4` variable? 

seems like we'd need to inform prod ops + teach them how to do this

Reply:
Yup, UA and GA4 tags will still fire separately, so there can be a time period where agencies can start to collect data on GA4 without losing their existing analytics, especially since there's no way to transfer existing UA data over to GA4

We were thinking of an additional settings field in the CMS to allow to configure GA4 in addition to the existing UA tag - this will be added in a future PR, and we're working out the details with design + pm/prod ops

Reply:
could i just clarify how this is set as the field isn't present atm; separately, do take note that UA4 should be sunsetted isn't it so we should just change the existing one to be GA4 rather than having both, no?

Reply:
Adding a PR for it in cms once design has been finalised! The triggering of UA right now is done via the existing `google_analytics.js` script

The reason why we're maintaining both for a while is that data is not transferrable between UA and GA4, so switching over immediately will break existing agencies implementation + it will take a while to collect enough data to be useful, so the ideal is to allow agencies to move off when they're comfortable instead